### PR TITLE
feat(default-theme): created and styled error list component

### LINKED ISSUE #1053: [FEATURE] [default-theme] Create helper for errorMessage
### Description
Should be created a helper for errorMessage which displays all errors from array nicely f.eg. list with errors.

### Acceptance criteria

- [ ] the appropriate component is created
- [ ] the component should be stateless
- [ ] all the form-errors notifications over the application should be replaced with this component

### PR DESCRIPTION
**Pull request by Malwurf**
closes #1053 

- We have created helper to get error messages as an array so here was created "styling component" screen below
- micro refactor on components where errors list exist
- replaced old errors with new ones

## Changes

Screenshot from 2020-09-01 17-33-48

### Checklist

- [ ] I followed [contributing](https://github.com/DivanteLtd/shopware-pwa/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: packages/default-theme/components/SwErrorsList.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<script>
+import SwAlert from "@shopware-pwa/default-theme/components/atoms/SwAlert"
+
+export default {
+  name: "SwErrorsList",
+  components: { SwAlert },
+  props: {
+    list: {
+      type: Array,
+      default: null,
```
Comment (mkucmus) [inline packages/default-theme/components/SwErrorsList.vue:20]:
```suggestion
      default: [],
```

File: packages/default-theme/components/SwErrorsList.vue [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<template>
+  <div v-if="list" class="errors-list-component">
+    <SwAlert :message="alertMessage" type="danger" />
```
Comment (mkucmus) [inline packages/default-theme/components/SwErrorsList.vue:3]: please use `{{ $t(' some string to be translated') }}` or `| translate` filter wherever it's possible ;)

File: packages/default-theme/components/SwErrorsList.vue [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  components: { SwAlert },
+  props: {
+    list: {
+      type: Array,
+      default: null,
+    },
+  },
+  data() {
+    return {
+      alertMessage: "Encountered problems:",
```
Comment (mkucmus) [inline packages/default-theme/components/SwErrorsList.vue:25]: not sure, but the translation should come from ` this.$i18n.t(` function

File: packages/default-theme/components/atoms/SwAlert.vue [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<template>
```
Comment (mkucmus) [inline packages/default-theme/components/atoms/SwAlert.vue:1]: :+1:

**Review by mkucmus — CHANGES_REQUESTED** [review]
I like the general idea of a centralized error notification component. Just a few little changes and will be ready :muscle:

File: packages/default-theme/components/SwErrorsList.vue (hunk 2 above)
Comment (Malwurf) [reply packages/default-theme/components/SwErrorsList.vue]: Ok, I moved that string from data object to template and added $t() function + translations in locales files.

File: packages/default-theme/components/SwErrorsList.vue (hunk 3 above)
Comment (Malwurf) [reply packages/default-theme/components/SwErrorsList.vue]: I removed that part of code :+1:

File: packages/default-theme/components/SwErrorsList.vue (hunk 1 above)
Comment (Malwurf) [reply packages/default-theme/components/SwErrorsList.vue]: Thanks for suggestion :+1:

**Review by akrajna — APPROVED** [review]

**Review by mkucmus — APPROVED** [review]
:+1:

**Pull request merged**